test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1432,6 +1432,27 @@ describe('StructureDefinitionExporter', () => {
     });
   });
 
+  it('should not apply a Reference FixedValueRule with invalid type and log an error', () => {
```

Comment:
The test description says that this logs an error, but nothing in the test case checks to see if an error is logged.

Reply:
Added the check for the error! Good catch!